demo.py

```diff
@@ -0,0 +1,17 @@
+# from deepstack_sdk import ServerConfig, Detection
```

Comment:
Please remove this file. Instead, update the existing test codes in `test/` and add the new parameters to them. You can set them all to `True` for all the samples (but the whoever view the sample will understand they can use the parameters to turn off some visualization)